src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DocumentMapping/AbstractEditMappingService.cs

```diff
@@ -23,28 +23,68 @@ internal abstract class AbstractEditMappingService(
 
     public async Task<WorkspaceEdit> RemapWorkspaceEditAsync(IDocumentSnapshot contextDocumentSnapshot, WorkspaceEdit workspaceEdit, CancellationToken cancellationToken)
     {
-        if (workspaceEdit.TryGetTextDocumentEdits(out var documentEdits))
+        // Handle DocumentChanges - iterate through TextDocumentEdits and modify them in-place.
```

Comment:
>// Handle DocumentChanges - iterate through TextDocumentEdits and modify them in-place. [](http://example.com/codeflow?start=8&length=87)

nit: would it make sense to change the signature of this method to make it more clear that it modifies the incoming workspaceEdit? The private method conveys this well, so maybe something like

```
public async Task RemapWorkspaceEditInPlaceAsync(IDocumentSnapshot contextDocumentSnapshot, WorkspaceEdit workspaceEdit, CancellationToken cancellationToken)
```

Reply:
That's very good feedback, but I don't like the "in place" in the name 📦 

But I'll make the methods void (or Task), and doc, and that should make it clear.

Reply:
Also completely removing the term "remap" from these files, as I don't know what it means. They weren't mapped before, so I don't know how they can be "re" mapped.